pkg/services/ngalert/notifier/testing.go

```diff
@@ -10,6 +10,7 @@ import (
 	"testing"
 	"time"
 
+	"github.com/grafana/alerting/images"
```

Comment:
wdyt about using an alias `alertingImages`? It will match the approach we follow with `alertingNotify`